.gitignore

```
@@ -8,3 +8,5 @@ node_modules
 # Local Netlify folder
 .netlify
 .DS_Store
+/prisma/docs
+prisma/credentials
```

Since these might be too personal, can remove.

```suggestion
.netlify
.DS_Store
```